plugins/bufbuild/connect-kotlin/v0.1.3/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.4
+FROM gradle:7.4.0-jdk17 AS build
+WORKDIR /app
+RUN git clone --depth 1 --branch v0.1.3 https://github.com/bufbuild/connect-kotlin.git
+WORKDIR /app/connect-kotlin
+RUN gradle --no-daemon -g $(pwd) protoc-gen-connect-kotlin:jar
```

Comment:
We should be able to update to pull the plugin from the Github release so we don't have to rebuild it here, but we might need to retry the release build first.